lib/api/channel.js

```diff
@@ -359,6 +359,59 @@ class Channel extends ApiResource {
       `/channels/${channelId}/featured-videos.json`,
       qs.stringify({video_ids: videoIds}))
   }
+
+  /**
+   * Getting the channel's recording status
+   * While broadcasting we can have the video it's recorded to
+   *
+   * @param {Number}   channelId
+   *
+   * @returns {Promise}
+   */
+  getRecordingStatus (channelId) {
+    return this.context.authRequest('get', `/channels/${channelId}/recorder.json`)
+  }
+
+  /**
+   * Set (start or stop) recording status for broadcasts
+   *
+   * @param {Number}   channelId
+   * @param {string}   command
+   *
+   * @returns {Promise}
+   */
+  setRecordingStatus (channelId, command) {
+    return this.context.authRequest('post', `/channels/${channelId}/recorder.json`,
+      qs.stringify({ command }));
+  }
+
+  /**
+   * set autorecord for broadcasts
+   *
+   * @param {Number}   channelId
+   * @param {string}   value - Possible values: disabled, private, public.
+   * If the value is disabled, no auto-recorded video gets created for the channel when broadcasting.
+   * If the value is private, then the new auto-record will be private by default.
+   * In case of public the created auto-records will be public by default.
```

Comment:
Please indent multi-line descriptions. Ensure no line is greater than 80 characters in length.

```
* @param {string} value - Possible values: disabled, private, public.
*               If the value is disabled, no auto-recorded video gets created for the channel
*               when broadcasting. If the value is private, then the new auto-record will be private
*               by default. In case of public the created auto-records will be public by default.
```